src/Common/tests/System/Security/Cryptography/AlgorithmImplementations/ECDsa/ECDsaTests.cs

```diff
@@ -517,6 +517,7 @@ public void Roundtrip_WithOffset(ECDsa ecdsa)
             Assert.True(ecdsa.VerifyData(halfData, dataSignature, HashAlgorithmName.SHA256));
         }
 
+#if netcoreapp // uses ECParameters not available in desktop.
```

Comment:
Yes I usually separate them but @bartonjs prefers this model because this are types that are going to be available on 4.7 and he told me it would be easier to bring the tests back to Desktop when that happens by just deleting the ifdefs + a lot of test projects would be affected. 

See: https://github.com/dotnet/corefx/pull/17463